README.md

```diff
@@ -40,169 +40,23 @@ driftctl tracks how well your IaC codebase covers your cloud configuration. drif
 ## Features
 
 - **Scan** cloud provider and map resources with IaC code
-- Analyze diff, and warn about drift and unwanted unmanaged resources
+- Analyze diffs, and warn about drift and unwanted unmanaged resources
 - Allow users to **ignore** resources
 - Multiple output formats
 
-## Documentation & support
+---
 
-- [Get started](https://driftctl.com/product/quick-tutorial/)
-- [User guide](doc/README.md)
-- [Discord](https://discord.gg/NMCBxtD7Nd)
+**[Get Started](https://driftctl.com/product/quick-tutorial/)**
 
-## Getting started
+**[Documentation](https://docs.driftctl.com)**
```

Comment:
It could be nice to have a direct link to installation page there

Reply:
Do you mean I need to add after `Documentation` a link to `Installation` ? Or simply change the URL of the `Documentation` to redirect to the `Installation` page ?

Reply:
I meant a new link after or maybe before documentation

Reply:
Done fca36e1